ably/auth_test.go

```diff
@@ -10,11 +10,11 @@ import (
 var _ = Describe("Auth", func() {
 	Describe("RequestToken", func() {
 		It("gets a token from the API", func() {
-			req := client.Auth.CreateTokenRequest()
+			req := client.Auth().CreateTokenRequest()
```

Comment:
Hum. I thought this was kind of in-line with the change needed for `Presence`. The only way for a property to be immutable outside the package you have to use accessors.

We discussed this for `Presence` to be `Presence()` instead, so if we enforce such a behaviour elsewhere, I would advise to be consistent with this one too.